### PR TITLE
[fix](nereids)single side info is missing when pushing min-max runtime filter into cte

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
select * from T1 join T2 where T1.x>T2.y
we have min-max rf(T2.y->T1.x), in which the min-max only have one side comparison
when pushing this type of rf into cte, the single side min (or max) is missing

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/IdGenerator.java [hunk 1]
```diff
@@ -34,4 +34,8 @@ public IdGenerator<IdType> resetId(int initialId) {
     }
 
     public abstract IdType getNextId();
+
+    public int getCurrentId() {
+        return nextId;
+    }
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/common/IdGenerator.java:40]: not used, remove it

Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/common/IdGenerator.java:40]: used for debug

**Review by morrySnow — DISMISSED** [review]

**Comment by englefly** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by LiBinfeng-01 — APPROVED** [review]

**Pull request merged**